### PR TITLE
Add new bee blocks to toolbox xml

### PR DESCRIPTION
**Pull request by ajpal**
Quick follow up to https://github.com/code-dot-org/code-dot-org/pull/39611
Adding the blocks to the xml here allows the blocks to be added to the start blocks or toolbox in edit mode
image

### CODE REVIEW

File: dashboard/app/models/levels/karel.rb [hunk 1]
```diff
@@ -122,6 +122,9 @@ def toolbox(type)
     <category name="Bee">
       <block type="maze_nectar"></block>
       <block type="maze_honey"></block>
+      <block type="bee_n_honey"></block>
+      <block type="bee_n_honey"></block>
```
Comment (epeach) [inline dashboard/app/models/levels/karel.rb:126]: Should one of those be `nectar`?

Comment (ajpal) [reply dashboard/app/models/levels/karel.rb:126]: whoops, yes good catch thanks!

Comment (ajpal) [reply dashboard/app/models/levels/karel.rb:126]: fixed:
image

**Review by epeach — APPROVED** [review]
:bee:

**Pull request merged**